frontend/src/views/admin/CategoryEdit.jsx

```diff
@@ -1,16 +1,20 @@
 import React from 'react'
 import { useState, useEffect } from 'react';
 import { Link } from 'react-router-dom';
-import { getData } from '../../api/mockApi';
+import axiosInstance from '../../api/api';

```

Comment:
rename the file name to BaseApi
```suggestion
import axiosInstance from '../../api/BaseApi';

```